src/video-interface.js

```diff
@@ -530,7 +532,9 @@ export const VideoServiceSignals = {
 
 /** @param {!AmpElement|!VideoOrBaseElementDef} video */
 export function delegateAutoplay(video) {
-  video.signals().signal(VideoServiceSignals.AUTOPLAY_DELEGATED);
+  whenUpgradedToCustomElement(video).then(el => {
```

Comment:
Shouldn't we also add this to `userInteractedWith` below?

Reply:
I tried it briefly and it seems to work but I'm not sure I understand all the edge cases there could be to make the change. :(